src/Pages/Home.tsx

```diff
@@ -1,16 +1,86 @@
-import React, { Fragment } from 'react';
-import { BoxStyle, TypographyStyle } from '../HomeStyle';
-import  TodoListComponent  from "../Component/TodoList";
+import React, { useState, Fragment } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { TodoList, ListedItem, DeleteEditButton, ParentBox, Heading } from '../Style/HomeStyle';
+import { deleteTodo, editTodo } from '../Store/Slices/todoSlice'
+import { RootState } from '../store';
+import { Alert, ListItemSecondaryAction, ListItemText } from '@mui/material';
+import LogoutTodoButton from '../Component/LogoutTodoButton';
+import InputTextTodo from '../Component/InputTextTodo';
+import AddButtonTodo from '../Component/AddButtonTodo';
+import UpdateButtonTodo from '../Component/UpdateButtonTodo';
 
 const Home: React.FC = () => {
+  const [todoText, setTodoText] = useState<string>('');
+  const [showAlert, setShowAlert] = useState<boolean>(false);
+  const [showUpdateButton, setShowUpdateButton] = useState<boolean>(false);
+  const [showAddButton, setShowAddButton] = useState<boolean>(true);
+  const dispatch = useDispatch();
+  const todoList = useSelector((state: RootState) => state.todos.todoList);
+
+  const handleDeleteButtonClick = (id: string) => {
+    dispatch(deleteTodo(id));
+  };
+  const [newId, setNewId] = useState<string>("");
```

Comment:
i dont think we need this , we can just use uuid

Reply:
it is just used for storing uuid of list, for update and delete use this stored uuid,